packages/react-scripts/config/utils/ensureSlash.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function ensureSlash(path, needsSlash) {
```

Comment:
So ideally Webpack configs shouldn't need `ensureSlash` because they should already have everything they need in the final form in the `paths`.